helix-view/src/editor.rs

```diff
@@ -61,6 +61,7 @@ pub struct Config {
     pub completion_trigger_len: u8,
     /// Whether to display infoboxes. Defaults to true.
     pub auto_info: bool,
+    pub hide_gitignore: bool,
```

Comment:
Let's match the name on walk builder: `git_ignore`. This also needs to be documented in the `book/`

Reply:
I think I have addressed these changes on commit [45c131c](https://github.com/helix-editor/helix/pull/988/commits/45c131cc605be96dbe40e119ca46c814ce9a604c).